lighthouse-treemap/app/styles/treemap.css

```
@@ -71,7 +96,28 @@ body {
 }
 
 .lh-treemap {
-  margin: 10px;
+  margin: 2px;
+}
+
+.lh-table {
+  overflow-y: hidden;
+}
+
```

```suggestion

.tabulator {
  background-color: #f3f3f3; /* flash to neutral gray while loading new rows */
  contain: strict;
}
```

contain for possible perf win.  also tbh it also works well on `.lh-treemap`.

---

done, and added contain to treemap.